build-info-extractor/src/main/java/org/jfrog/build/extractor/clientConfiguration/client/artifactory/services/GetRepositoriesKeys.java

```diff
@@ -37,8 +38,9 @@ public HttpRequestBase createRequest() {
 
     @Override
     protected void setResponse(InputStream stream) throws IOException {
-        GetRepositoriesKeyResponse localRepositories = getMapper(true).readValue(stream, GetRepositoriesKeyResponse.class);
-        result = localRepositories.getRepositoriesKey();
+        List<GetRepositoriesKeyResponse> keys = getMapper(true).readValue(stream, new TypeReference<List<GetRepositoriesKeyResponse>>() {
+        });
```

Comment:
Let's do the same as we do in RetrievePatternArtifacts
```suggestion
        List<GetRepositoriesKeyResponse> keys = getMapper(true).readValue(stream, TypeFactory.defaultInstance().constructCollectionLikeType(List.class, GetRepositoriesKeyResponse.class));
```